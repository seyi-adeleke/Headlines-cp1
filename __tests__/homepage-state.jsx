import React from 'react';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';

import homepage from '../src/components/HomePage.jsx';

describe('homepage', () => {
  it('renders correctly', () => {
    const rendered = renderer.create(
      <homepage />,
    );
    expect(rendered.toJSON()).toMatchSnapshot();
  });
});



