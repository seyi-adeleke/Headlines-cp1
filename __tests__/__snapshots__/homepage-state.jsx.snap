// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`homepage renders correctly 1`] = `<homepage />`;

exports[`login renders correctly 1`] = `<login />`;
